rocana-release.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
```

Comment:
I've had a lot of bad luck with the maven release plugin especially on our Jenkins so I've been doing this script based approach for a bunch of our projects.